scylla/src/execution/mod.rs

```diff
@@ -6,12 +6,14 @@
 //!     - retries,
 //!     - speculative execution,
 //! - cluster-side request tracing,
+//! - driver metrics,
 //! - request execution history,
 //! - TODO.
 
 pub mod execution_profile;
 pub mod history;
 pub mod load_balancing;
+pub(crate) mod metrics;
 pub mod retries;
 pub mod speculative_execution;
```

Comment:
🌱 Internal code refers to metrics using the full path (`crate::execution::metrics::Metrics`) which is not accessible for the user. I believe we should make this module pub, and the current `scylla::Metrics` reexport should become `scylla::prelude::Metrics`. But this is something for a follow-up.

Reply:
I made the module `pub` and removed the re-export.